index.html

```diff
@@ -11,7 +11,7 @@
   <link rel="stylesheet" href="resources/styleheets/main.css">
   <script src="resources/js/main.js"></script>
 </head>
-<body>
+<body class="base">
```

Comment:
it is necessary add base class in the body? because the class login doesn't modify the style that the base have, so the base can be call raw as body element and just calling the login class in the login.html will apply the content you want